src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/Controllers/Domain/TeamsController.cs

```diff
@@ -171,21 +180,35 @@ public JsonResult Edit(TeamViewModel teamViewModel)
             {
                 try
                 {
-                    var roster = teamViewModel.Roster.Select(t => t.ToDomain()).ToList();
+                    if (teamViewModel.AddedPlayers.Count > 0)
+                    {
+                        var playersIdsToAddToTeam = teamViewModel.AddedPlayers
+                            .Where(x => x.Id > 0)
+                            .Select(x => new PlayerId(x.Id))
+                            .ToList();
+
+                        var playersToAddToTeam = teamViewModel.AddedPlayers
+                            .Where(x => x.Id == 0)
+                            .Select(x => x.ToCreatePlayerDto())
+                            .ToList();
+                        
+                        playersIdsToAddToTeam.AddRange(_playerService.CreateBulk(playersToAddToTeam)
```

Comment:
You still can call it with empty collection.

Reply:
yes, I`ve changed this because there were 3 nested controll statements, and sonar errored this, it is okay, there will be no exception when we call that with empty list